python/ray/autoscaler/aws/development-example.yaml

```diff
@@ -80,6 +81,9 @@ file_mounts: {
 
 # List of shell commands to run to set up nodes.
 setup_commands:
+    # Consider uncommenting these if you run into dpkg locking issues
+    # - sudo pkill -9 apt-get || true
+    # - sudo pkill -9 dpkg || true
```

Comment:
why not also 
`- sudo dpkg --configure -a` here?

Reply:
Done